demos/mqtt/mqtt_demo_subscription_manager/mqtt_demo_subscription_manager.c

```diff
@@ -322,22 +322,24 @@ static uint32_t generateRandomNumber();
  * back-off period is reached or the number of attempts are exhausted.
  *
  * @param[out] pNetworkContext The output parameter to return the created network context.
+ * @param[in] pMqttContext MQTT context pointer.
  *
  * @return EXIT_FAILURE on failure; EXIT_SUCCESS on successful connection.
  */
-static int connectToServerWithBackoffRetries( NetworkContext_t * pNetworkContext );
+static int connectToServerWithBackoffRetries( NetworkContext_t * pNetworkContext,
+                                              MQTTContext_t * pMqttContext );
 
 /**
  * @brief A function that connects to the MQTT broker,
  * subscribes to a topic, publishes to the same topic
  * MQTT_PUBLISH_COUNT_PER_LOOP number of times, and verifies if it
```

Comment:
(Minor) The function does not make an MQTT connection anymore
```suggestion
 * @brief A function that uses the passed MQTT connection to
 * subscribe to a topic, publish to the same topic
 * MQTT_PUBLISH_COUNT_PER_LOOP number of times, and verify if it
```